docs/releases/5.1.txt

```diff
@@ -349,3 +349,9 @@ to remove usage of these features.
 * The ``DEFAULT_FILE_STORAGE`` and ``STATICFILES_STORAGE`` settings is removed.
 
 * The ``django.core.files.storage.get_storage_class()`` function is removed.
+
+Bugfixes
+========
+
+* Made ``loaddata`` skip loading data when the table is in a different db in a
+  multi-db setup which used to throw an error while doing so. (:ticket:`34907`).
```

Comment:
We don't document bugfixes in the main release notes, please revert.

Reply:
Sure @felixxm, Shall revert, what'd be the correct place for this if documentation is needed or we can skip it altogether?

Reply:
Sure @felixxm, Shall revert, what'd be the correct place for this if documentation is needed or we can skip it altogether? I took the inspiration from [docs/releases/4.2.8.txt](https://github.com/django/django/blob/main/docs/releases/4.2.8.txt)

Reply:
We document regressions in bugfix releases. This is not the case here, so we don't need release notes here.

Reply:
Reverted!